navigation.js

```diff
@@ -26,24 +26,27 @@ document.addEventListener('click', (event) => {
   }
 });
 
-// Get the popup element
 const popup = document.querySelector('.popup');
-
-// Get the button that triggers the popup
-const btn = document.querySelector('.popup-btn');
-
-// Get the close button
+const btn = document.querySelectorAll('.popup-btn');
 const close = popup.querySelector('.close');
 
-// When the user clicks the button, display the popup
-btn.addEventListener('click', () => {
-  popup.style.display = 'block';
-});
+const popGenrato = () => {
+  const popup = document.querySelector('.popup');
+  const btn = document.querySelectorAll('.popup-btn');
+  const close = popup.querySelector('.close');
+  btn.forEach((btns) => {
+    btns.addEventListener('click', () => {
+      popup.style.display = 'block';
+    });
+  });
+  close.addEventListener('click', () => {
+    popup.style.display = 'none';
+  });
+};
```

Comment:
- [ ] Just as I said in the last review, the cards do not have a close button on mobile as well as the live buttons are not visible when rendered and have to be refreshed to be deleted. please fix this to give a better user experience, and also to follow project guidelines 

## View:
![image](https://user-images.githubusercontent.com/50990925/224525732-734d5834-f6ab-43d5-9b3e-cefd66c0e0ed.png)

## Figma:
![image](https://user-images.githubusercontent.com/50990925/224525764-9c1e32a9-b20a-4caf-8d2d-005eca6114af.png)

Reply:
@Shedrack-Sunday  Man come they have it!

Reply:
<img width="822" alt="Screenshot 2023-03-12 at 12 22 15 AM" src="https://user-images.githubusercontent.com/88105477/224526022-d
<img width="822" alt="Screenshot 2023-03-12 at 12 22 15 AM" src="https://user-images.githubusercontent.com/88105477/224526027-dfbd7862-c2c7-4fda-b9f7-a5c7b0e9065f.png">
86a3fb2-d75c-4118-ac05-56b3c33168cc.png">

Reply:
Desktop version!! 

<img width="1410" alt="Screenshot 2023-03-12 at 12 21 54 AM" src="https://user-images.githubusercontent.com/88105477/224526160-827d1e4f-27a0-4201-82ec-d1ec08049b5a.png">

Reply:
@Shedrack-Sunday I'm getting this in both scenarios.

Reply:
@jlberbesi please see my detailed review on where the buttons should be placed on the desktop version. This is not the same as given on the template. Check my review above. If the review is not satisfactory, please send for a second opinion. Let me know if you need clarity. 

You can also as for second opinion